src/common/system/proc_parser_test.cc

```diff
@@ -299,7 +299,7 @@ TEST_F(ProcParserTest, read_proc_fd_link) {
     ASSERT_OK(
         fs::CreateSymlinkIfNotExists("/dev/null", GetPathToTestDataFile("testdata/proc/123/fd/0")));
     ASSERT_OK(
-        fs::CreateSymlinkIfNotExists("/foobar", GetPathToTestDataFile("testdata/proc/123/fd/1")));
+        fs::CreateSymlinkIfNotExists("./foobar", GetPathToTestDataFile("testdata/proc/123/fd/1")));
```

Comment:
The test is currently disabled because symlink don't work with bazel. (see comment above function proto

Reply:
This test is definitely executing in your build. The Buildbuild link for each build shows this is running:

![screen](https://github.com/user-attachments/assets/799c7880-3c5d-4944-bd58-3f15314f1346)

This comment is out of date as of [this commit](https://github.com/pixie-io/pixie/commit/00ca3803aafc7066aef9797ad442d3296c35d2ae#diff-5dc6a3f4b9eb4767d7b2e4b33e084ff0b7f153b09e6dc2bdb7cd68ccbad9bcb3R134). We fixed the test but didn't realize the comment about was no longer relevant.

Please remove this comment from above the test.